dotcom-rendering/src/web/components/Callout/CalloutDescription.tsx

```diff
@@ -0,0 +1,41 @@
+import { css } from '@emotion/react';
+import { textSans, space, body } from '@guardian/source-foundations';
+import { decidePalette } from '../../lib/decidePalette';
+
+const descriptionStyles = (format: ArticleFormat) =>
+	css`
+		a {
+			color: ${decidePalette(format).text.richLink};
+			border-bottom: 1px solid ${decidePalette(format).text.richLink};
+			text-decoration: none;
+		}
+		${textSans.small()}
+		padding-bottom: ${space[4]}px;
+		${body.medium()}
+
+		p {
+			margin-bottom: ${space[3]}px;
+		}
+	`;
+
+interface Props {
+	description: string;
+	format: ArticleFormat;
+}
+
+export const CalloutDescription = ({ description, format }: Props) => (
+	<div css={descriptionStyles(format)}>
+		<div dangerouslySetInnerHTML={{ __html: description }}></div>
```

Comment:
The description we get in the JSON contains some html tags, hence we set html here

Reply:
Looks like the `sanitize-html` library is installed. We should use it to sanitise the description to be safe.

Reply:
Thanks - implemented